reuse.go

```diff
@@ -0,0 +1,190 @@
+package libp2pquic
+
+import (
+	"errors"
+	"net"
+	"sync"
+
+	srcs "github.com/lnykww/go-src-select"
```

Comment:
I've forked this to github.com/libp2p/go-src-select I can also delete that and we can transfer it if you'd like.

Could you add a LICENSE? Ideally Apache 2.0 + MIT. Take a look at the COPYRIGHT, LICENSE-MIT, and LICENSE-APACHE files in https://github.com/ipfs/go-ds-crdt/commit/d51c9f1306f317b77e7c314113b8643a0a471b82.

Note: This project is still copyright _you_ (well, you and whoever else ends up contributing to it).

Reply:
Done

Reply:
Thanks!